code/modules/surgery/coronary_bypass.dm

```diff
@@ -127,3 +127,18 @@
 		var/obj/item/bodypart/target_bodypart = target_human.get_bodypart(target_zone)
 		target_bodypart.adjustBleedStacks(30)
 	return FALSE
+
+/datum/surgery/coronary_bypass/mechanic
```

Comment:
modularize (just move this to `monkestation/code/modules/surgery/coronary_bypass.dm`)